ownCloud/Client/ClientItemCell.swift

```diff
@@ -90,6 +101,34 @@ class ClientItemCell: ThemeTableViewCell {
 			self.accessoryType = .none
 		}
 
+		if item.thumbnailAvailability != .none {
+			let displayThumbnail = { (thumbnail: OCItemThumbnail?) in
+				thumbnail?.requestImage(for: thumbnailSize, scale: 0, withCompletionHandler: { (thumbnail, error, _, image) in
+					if error == nil {
+						DispatchQueue.main.async {
+							if image != nil {
+								if self.item?.versionIdentifier == thumbnail?.versionIdentifier {
```

Comment:
Also in Swift you can chain multiple if statements with a `,`. Don't know which option is more clear to you if the one with three if statements or chain them like: 
```Swift
if error == nil, image != nil, self.item?.versionIdentifier == thumbnail?.versionIdentifier {
  OnMainThread {
       self.iconView.image = image
   }
}
```

Reply:
Good idea. I made that change.